pages/_document.tsx

```diff
@@ -3,6 +3,7 @@ import Document, { Html, Head, Main, NextScript } from 'next/document';
 import { extractCritical } from '@emotion/server';
 
 export default class MyDocument extends Document {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
could you follow this for fixing the eslint error
https://stackoverflow.com/questions/67087999/how-to-properly-type-the-document-tsx-file-from-next-js